bin/sozo/src/commands/calldata_decoder.rs

```diff
@@ -150,6 +174,7 @@ fn decode_inner(item: &str) -> DecoderResult<Vec<Felt>> {
             "u256" => U256CalldataDecoder.decode(value)?,
             "str" => StrCalldataDecoder.decode(value)?,
             "sstr" => ShortStrCalldataDecoder.decode(value)?,
+            "-" => SignedIntegerCalldataDecoder.decode(value)?,
```

Comment:
```suggestion
            "int" => SignedIntegerCalldataDecoder.decode(value)?,
```
Use `int` instead, I didn't think about clap that will try to consume the `-`, hence not working.